### PR TITLE
Updated the changes.

### PR DESCRIPTION
**Pull request by ritikrana4**
hey @MadaZZ , 
I have corrected what you asked for. Please see if any other changes are required.

### CODE REVIEW

File: 1 Foundations/this.js [hunk 1]
```diff
@@ -1,37 +1,160 @@
-// Definition of 'this'
-// 'this' is an object that the function is a property of.
```
Comment (MadaZZ) [inline 1 Foundations/this.js:2]: This definition should have been let be. This is something very short and explains a lot.

File: 1 Foundations/this.js [hunk 2]
```diff
@@ -1,37 +1,160 @@
...
+// Lets see in detail the different ways of evaluating this keyword.
 
-/* Why is 'this' important?*/
+// 1.Implicit Binding
 
+const user = {
+  name: "ritik",
+  age: "22",
+  welcome() {
+    alert(`Hello ${this.name} `);
```
Comment (MadaZZ) [inline 1 Foundations/this.js:15]: Replace the alert with logs. 
That way, the code works properly in `repl.it` and also using node via terminal.

**Review by MadaZZ — CHANGES_REQUESTED** [review]

**Review by MadaZZ — APPROVED** [review]
Looks good

**Comment by ritikrana4** [comment]
@MadaZZ  great 👍

**Pull request merged**